package.json

```diff
@@ -12,7 +12,7 @@
   },
   "homepage": "https://github.com/vent-io/project-template#readme",
   "workspaces": [
-    "packages/*"
+    "src/*"
```

Comment:
A bit of a hack, right?

If I understand correctly, this repo corresponds to a single Python package. With the `src` directory and `pyproject.toml` in the repo root, we cannot have multiple packages anyway, if [I read this right](https://discuss.python.org/t/multiple-packages-from-same-src-using-pyproject-toml/30998/13). So all monorepo concerns are out of the question for this repo, and a `cd src && ./test.sh` or whatever `lerna` is doing doesn't make sense (`pyproject.toml` and `pdm.lock` wouldn't be found).

Reply:
It was planned to set it up in a monorepo way, so other oss packages could be added later. I will talk to @NicolasBogun